var/spack/repos/jcsda-emc/packages/jedi-cmake/package.py

```diff
@@ -23,9 +23,6 @@ class JediCmake(CMakePackage):
         preferred=True,
         submodules=True,
     )
-    version("1.3.0", sha256="3e92339df858e9663b2cdd9f7bb7e56d18098e9c60606fe7af9e5f5911e5ca55")
-    version("1.2.0", sha256="eb9f1c403d1b43a90a5e774097382b183d56d5b40a1204b51af2da8db1559b21")
-    version("1.1.0", sha256="f1fe41eb5edd343bdf57eb76bea6d1b9f015878f0a9d0eb1e9dba18b903d3b35")
-    version("1.0.0", sha256="d773a800350e69372355b45e89160b593818cd438a86925b8a689c47996a0b9a")
+    version("1.3.0", commit="729a9b2ec97a7e93cbc58213493f28ca11f08754")
```

Comment:
I'm not asking to change this, but for my own edification is this change necessary because of the switch to using a submodule in jedi-cmake? Thanks!

Reply:
Yes, it is. Plus we don't ever plan to build older versions of jedi-cmake.

Reply:
Sounds good. Thanks!